src/components/IPScan/ImportResultSearch/LoadedFileDialog/index.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+
+import { connect } from 'react-redux';
+import { importJobFromData } from 'actions/creators';
+import id from 'utils/cheap-unique-id';
+
+import Modal from 'components/SimpleCommonComponents/Modal';
+import IPScanVersionCheck from 'components/IPScan/IPScanVersionCheck';
+import NucleotideCheck from 'components/IPScan/NucleotideCheck';
+import Button from 'components/SimpleCommonComponents/Button';
+
+const isValid = (fileObj: Record<string, unknown>) => {
+  return (
+    typeof fileObj === 'object' &&
+    'results' in fileObj &&
+    'interproscan-version' in fileObj
+  );
+};
+
+// TODO: remove this types once the nucleotide file logic is restored
```

Comment:
What do you mean, by "restored"?

Reply:
Nah, that was a note to myself, while I was in the middle of the migration, the logic was restored, so now I can delete that :)